components/src/dynamo/rl_admin/service.py

```
@@ -0,0 +1,434 @@
+    """Encode a single token's text to its ID using a lightweight tokenizer."""
+    global _tokenizer_for_fallback
+    if _tokenizer_for_fallback is None:
+        try:
+            from transformers import AutoTokenizer
+            model = MODEL_NAME or os.getenv("MODEL_NAME", "")
+            if model:
+                _tokenizer_for_fallback = AutoTokenizer.from_pretrained(
+                    model, trust_remote_code=True
+                )
+        except Exception:
+            pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)